infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/event/PodEvent.java

```diff
@@ -11,22 +11,23 @@
 package org.eclipse.che.workspace.infrastructure.kubernetes.namespace.event;
 
 import java.util.Objects;
+import org.eclipse.che.commons.annotation.Nullable;
 
 /**
  * The event that should be published when container change occurs, e.g. image pulled, container
```

Comment:
Please fix java docs. Now it should be published even when no container changes occur. It would be useful to add an example of a case when pod event is published.